packages/builder/src/components/design/settings/controls/DataSourceSelect/DataSourceSelect.svelte

```diff
@@ -116,8 +115,11 @@
       }
     })
   $: fields = bindings
-    .filter(x => arrayTypes.includes(x.fieldSchema?.type))
```

Comment:
This was causing the attachments and array fields to not be displayed properly on the dataprovider selector